lib/stripeHelpers.js

```diff
@@ -3,7 +3,7 @@ const stripe = Stripe(process.env.STRIPE_SECRET_KEY)
 
 const findOrCreateCustomer = async ({ email, name, source, meta }) => {
   const { data } = await stripe.customers.list({ email })
-  if (data.length && source) {
+  if (data.length) {
```

Comment:
@OVECJOE This was creating a new customer each donation even if the email already existed. this is because stripe sessions are now created without "sources"